examples/gesture_recognizer/android/app/src/main/res/values/colors.xml

```diff
@@ -28,4 +28,8 @@
     <color name="icActive">#FFFFFFFF</color>
     <color name="icFocused">#DDFFFFFF</color>
     <color name="icPressed">#AAFFFFFF</color>
+    <color name="htwGrey">#AFAFAF</color>
```

Comment:
Another reason I'm guessing you're using this for a school project - changing the branding :)